OpenSwuNG/Classes/Schedule(课程表)/Controller/SWUScheduleViewController.m

```diff
@@ -0,0 +1,93 @@
+//
+//  SWUScheduleViewController.m
+//  OpenSwuNG
+//
+//  Created by 张俊 on 2019/2/16.
+//
+
+#import "SWUScheduleViewController.h"
+
+@interface SWUScheduleViewController ()
+
+@end
+
+@implementation SWUScheduleViewController
+
+- (void)viewDidLoad {
+    [super viewDidLoad];
+    self.view.backgroundColor = [UIColor whiteColor];
+    
+    // Uncomment the following line to preserve selection between presentations.
+    // self.clearsSelectionOnViewWillAppear = NO;
+    
+    // Uncomment the following line to display an Edit button in the navigation bar for this view controller.
+    // self.navigationItem.rightBarButtonItem = self.editButtonItem;
+}
+
+#pragma mark - Table view data source
+
+- (NSInteger)numberOfSectionsInTableView:(UITableView *)tableView {
+#warning Incomplete implementation, return the number of sections
```

Comment:
`#warning Incomplete implementation, return the number of sections`
Remove this.